models/env_service_model.py

```diff
@@ -193,3 +195,14 @@ def get_user_input_api_keys():
                 return False
         except:
             return False
+
+    @staticmethod
+    def get_user_key_db_path() -> Union[Path, None]:

```

Comment:
These changes look good! But can you also update the `sample.env` and the README slightly if you don't mind?

Reply:
Sure